serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java

```diff
@@ -0,0 +1,33 @@
+package no.difi.meldingsutveksling.serviceregistry.logging;
+
+import net.logstash.logback.marker.LogstashMarker;
+import net.logstash.logback.marker.Markers;
+
+public class SRMarkerFactory {
+
+    private SRMarkerFactory() {}
+
+    private static final String REMOTE_IP = "remote_ip";
+    private static final String CLIENT_ID = "client_id";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1213)